plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/ApplicationSourceListPage.java

```diff
@@ -615,36 +624,36 @@ private TreeViewer createServerTemplatesViewer(Composite parent, final Text temp
 		TreeViewer viewer = new TreeViewer(parent, SWT.BORDER | SWT.SINGLE | SWT.V_SCROLL | SWT.H_SCROLL);
 		IListProperty childrenProperty = new MultiListProperty(
 				new IListProperty[] {
-						BeanProperties.list(ITemplateListPageModel.PROPERTY_TEMPLATES),
+						BeanProperties.list(IApplicationSourceListPageModel.PROPERTY_APP_SOURCES),
 						BeanProperties.list(ObservableTreeItem.PROPERTY_CHILDREN) });
 		ObservableListTreeContentProvider contentProvider = new ObservableListTreeContentProvider(
 				childrenProperty.listFactory(), null);
 		viewer.setContentProvider(contentProvider);
 		viewer.setLabelProvider(new ObservableTreeItemStyledCellLabelProvider());
```

Comment:
@jcantrill , please set to inner OpenShiftExplorerLabelProvider explorerLabelProvider label limit 100. Maybe it is better to do by adding a method, or it can be as well done just inside ObservableTreeItemStyledCellLabelProvider, since it is currently used in wizards only.

Auto-adjusting abbreviation inside available width can be not a good idea (though worth to study), because view asks texts from label provider on refresh, not on resize, and calling refresh of view on every resize event may cause performance issues. So, if we just have an idea of the optimal wizard size, we may fit label limit to it. The default 60 is of course too small.